docs/development.md

```diff
@@ -22,13 +22,27 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
 
 ### With Eclipse
 
-* To start the dev server, right-click on the project folder and choose `Run As → Web Application`.<br>
-  After some time, you should see this message (or similar) on the Eclipse console: `Dev App Server is now running`.
-  The dev server URL will be given at the console output, e.g `http://localhost:8888`.
+#### Starting the dev server
 
-* To stop the dev server, click the "Terminate" icon on the Eclipse console.
+Right-click on the project folder and choose `Run As → Web Application`.<br>
+After some time, you should see this message (or similar) on the Eclipse console: `Dev App Server is now running`.
+The dev server URL will be given at the console output, e.g `http://localhost:8888`.
 
-### Outside Eclipse
+#### Stopping the dev server
+
+Click the "Terminate" icon on the Eclipse console.
+
+### With IntelliJ
+
+#### Starting the dev server
+
+Ensure `Dev Server` is selected in the dropdown menu next to the `Play` icon in the toolbar. Click the `Play` icon.
```

Comment:
It might be better to have directions to use the menu bar, instead of the toolbar (iirc the toolbar is not enabled by default)